var/spack/repos/builtin/packages/py-scipy/package.py

```diff
@@ -139,7 +139,7 @@ class PyScipy(PythonPackage):
     # https://github.com/mesonbuild/meson/pull/10909#issuecomment-1282241479
     # Intel OneAPI ifx claims to support -fvisibility, but this does not work.
     # Meson adds this flag for all Python extensions which include Fortran code.
-    conflicts("%oneapi", when="@1.9:")
+    conflicts("%oneapi@:2023.0.0", when="@1.9:")
```

Comment:
Resolved in 2023.1.0 or in 2023.0.1? Might want:
```suggestion
    conflicts("%oneapi@:2023.0", when="@1.9:")
```

Reply:
Yes, thank you!